README.md

```diff
@@ -24,9 +24,11 @@ released version of Riot:
 Releases are signed by PGP, and can be checked against the public key
 at https://riot.im/packages/keys/riot.asc
 
-Note that Chrome does not allow microphone or webcam access for sites served
-over http (except localhost), so for working VoIP you will need to serve Riot
-over https.
+Note that for the security of your chats will need to serve Riot
+over HTTPS. Mayor browsers also do not allow you to use VoIP/video
+chats over HTTP, as WebRTC is only usable over HTTPS.
+An exception of this is localhost, which usually is alos trusted in
```

Comment:
Typo "alos". Propably should mean "also" but I think it is not needed in the sentence at all